app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -48,10 +49,20 @@ class TSnePlotViewRoute extends React.Component {
   }
 
   render() {
+
+    console.log(this.props)
+
     const { location, match, history } = this.props
     const { atlasUrl, suggesterEndpoint, defaultPlotMethodAndParameterisation } = this.props
     const { species, experimentAccession, accessKey, ks, ksWithMarkerGenes, plotTypesAndOptions, metadata, anatomogram } = this.props
     const search = URI(location.search).search(true)
+    if(species === `homo sapiens` && Object.keys(anatomogram).length >0){
```

Comment:
Please add a space after `>` for better coding style.

Reply:
Also we would use spece before `(` and after `)` with if condition. Please fix the same code format like this.

Reply:
Many `;` has not been removed in this file at least, please double check for all files.